torch/testing/_internal/distributed/_sharded_tensor/__init__.py

```diff
@@ -72,14 +72,18 @@ def assert_sharded_tensor_equal(self, st1, st2):
         self.assertEqual(len(st1.remote_shards()), len(st2.remote_shards()))
 
 # wrapper to initialize comms (processgroup + rpc)
-def with_comms(init_rpc=True, backend="nccl"):
+def with_comms(_func=None, *, init_rpc=True, backend="nccl"):
```

Comment:
Wondering if we can just do what `dist_init` does: https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/dist_utils.py#L20 to keep wrappers like these consistent.